scripts/tests

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Did you mean to commit this file?

If yes, is there a reason why you choose to `led in -s php80` and not `... php81`?

Reply:
Yes, I mean to commit this file. It's a dev repository, I (and anyone in PrestaConcept) want to feel comfortable when pulling it.

But there is no reason at using 8.0 container at this point, I guess I picked it because it is "stable" but there is no real reason behind this